docs/reference/bpmn-processes/message-events/message-events.md

```diff
@@ -49,6 +47,31 @@ The `correlationKey` is an expression that usually [accesses a variable](/compon
 
 To correlate a message to the message event, the message is published with the defined name (e.g. `Money collected`) and the **value** of the `correlationKey` expression. For example, if the process instance has a variable `orderId` with value `"order-123"`, the message must be published with the correlation key `"order-123"`.
 
+## Message throw events
+
+A process can contain intermediate message throw events or message end events to model the
+publication of a message to an external system, for example, to a Kafka topic.
+
+At the moment, intermediate message throw events and message end events behave exactly
+like [service tasks](../service-tasks/service-tasks.md) or [send tasks](../send-tasks/send-tasks.md)
+, and have the same job-related properties (e.g. job type, custom headers, etc.). The message throw
+events and the tasks are based on jobs
+and [job workers](../../../components/concepts/job-workers.md). The differences between the message
+throw events and the tasks are the visual representation and the semantics for the model. Read more
+about the job properties [here](../send-tasks/send-tasks.md#defining-a-task).
```

Comment:
❓ We reference the send-tasks page here for the job properties. On that page is a reference to job type, task headers and variable mappings. However, it is not mentioned that these are job properties. Maybe we could reference them to the Job workers page which contains a clear list of job properties? The job worker page also mentions the key property.